app/services/csv_exporter.rb

```diff
@@ -74,4 +74,8 @@ def to_csv
   def dequote_and_truncate(text)
     text.tr('"', '').tr("'", '')[0..4000]
   end
+
+  def humanize_boolean(boolean)
+    boolean ? 'Yes' : nil
```

Comment:
should this not be 'Yes'/'No' ?

Reply:
Looking at what we have currently we tend to leave the cell blank if it's no. The ticket too just said yes if true. Together I guessed that nil was the best choice